config/_default/menus/menus.en.yaml

```diff
@@ -3244,21 +3244,26 @@ main:
     parent: rum_explorer
     identifier: rum_explorer_watchdog_insights
     weight: 908
+  - name: Funnel Analysis
+    url: real_user_monitoring/funnel_analysis
+    parent: rum
+    identifier: rum_funnel_analysis
+    weight: 10
   - name: Generate Metrics
     url: real_user_monitoring/generate_metrics
     parent: rum
     identifier: rum_generate_metrics
-    weight: 10
+    weight: 11
   - name: Connect RUM and Traces
     url: real_user_monitoring/connect_rum_and_traces
     parent: rum
     identifier: rum_connect_rum_and_traces
-    weight: 11
+    weight: 12
   - name: Error Tracking
     url: real_user_monitoring/error_tracking/
     parent: rum
     identifier: rum_error_tracking
-    weight: 12
+    weight: 13
```

Comment:
not sure why github won't let me comment on the relevant line, but the RUM Guides item (the one with `identifier: rum_guides` also needs its weight changed to `14`